server/controllers/categorised/employees.js

```diff
@@ -1,7 +1,37 @@
+/**
+* The /employees HTTP API endpoint
+*
+* @module categorised/employees
+*
+* @description This controller is responsible for implementing all crud on the
+* employees table through the `/employees` endpoint.
+*
+* @requires lib/db
+* @requires lib/util
+* @requires lib/sanitize
+*
+* NOTE: This api does not handle the deletion of employees because
+* that subject is not in the actuality.
+*/
+
+'use strict';
+
 var db = require('./../../lib/db');
 var util = require('./../../lib/util');
 var sanitize = require('./../../lib/sanitize');
```

Comment:
Is sanitize ever actually used?  It doesn't look like it is - you might want to remove it.